Configuration/Geometry/python/GeometryRecoDB_cff.py

```diff
@@ -17,8 +17,7 @@
 from Geometry.CaloEventSetup.EcalTrigTowerConstituents_cfi import *
 from Geometry.EcalMapping.EcalMapping_cfi import *
 from Geometry.EcalMapping.EcalMappingRecord_cfi import *
-from Geometry.HcalCommonData.hcalDDDSimConstants_cfi import *
-from Geometry.HcalCommonData.hcalDDDRecConstants_cfi import *
+from Geometry.HcalCommonData.hcalDBConstants_cff import *
```

Comment:
@cvuosalo - the name assumes that they are taken from a DB.

Reply:
@ianna Some of the names of these HCal components are probably historical and may be a bit confusing. `HcalDDDRecConstantsESModule` reads from either the DB or XML to create the `HcalDDDRecConstants`. Similarly, `HcalDDDSimConstantsESModule` reads from DB or XML to create `HcalDDDSimConstants`. The `DDD` in these names might seem to imply they work only for DDD XML, but, in fact, they are used for both DDD and DD4hep and come from DB or XML.